pyvista/plotting/plotting.py

```diff
@@ -20,6 +20,7 @@
 import matplotlib
 import numpy as np
 import scooby
+from vtkmodules.vtkRenderingOpenGL2 import vtkOpenGLTexture
```

Comment:
If one of the goals of this PR is to allow users to avoid importing from any `vtkmodules.vtkRenderingOpenGL2`, I think these additions will still cause this import during `import pyvista`.

Reply:
One of the goals of this PR is to start to consider moving imports out of `_vtk` and to import when needed within methods.

While going through this PR, I realize that many of the imports are needed when creating classes, which makes it difficult to lazy import classes like `vtkCharts`, which isn't widely used.

I think I'm going to revert the last two and focus on just reader imports at the moment.

Reply:
Reverted in [2a3bed3](https://github.com/pyvista/pyvista/pull/4337/commits/2a3bed339553482d883289df885a7e4be8b59bfd).

Reply:
I'm not against moving the imports like this, but it will require some discussion on best path forward if/whether we want to avoid import unless used.  I'm +1 on your suggested strategy for this PR.